docs/20_tutorial.md

```diff
@@ -1,64 +1,245 @@
+
+Check that the information on the review screen reflects the 
+Audit Board's interpretation of the ballot. Note that in case of 
+an overvote, the review screen will say, simply, "overvote".
+
+Note: do *not* use your browser's "back" arrow during the audit process.
+If the review screen does not match the Audit Board's interpretation,
+click "Back".
+
+If the review screen matches the Audit Board's interpretation,
+click "Submit and Next Ballot". This submission is final. There 
+is no way to revise a ballot interpretation once it has been submitted
+from the Review Screen.
+
+# Comments and Questions on Draft
+
+***should the run book deal with the SOS establishing the risk
+limit, or any of their pre-audit responsibilities like the selection
+of which races to audit? (SS says, yes, briefly, in the context of 
+what needs to be entered into the tool.)
 
 ***should the run book cover the preparation/creation of these artifacts
-they’ll be uploading?
+they’ll be uploading? SS says: no, but should specify requirements for 
+these artifacts.
 
 Description of the process of generating the random seed, perhaps with a
 photograph of what the process will look like with folks looking on.
+***SS says: out of scope for the run book
 
 ***Will the risk limit be auto populated to the counties’ dashboards, or
-will they be instructed to enter it themselves?
+will they be instructed to enter it themselves? SS says: From my 
+understanding of the process, Counties don't actually need to know the 
+risk limit. If they want it, they should be able to find it on the public dash.
 
 ***should more detailed instructions/descriptions for the retrieving of
 ballots, and the rules about interpretation of ballot marks, be included
-in the run book?
+in the run book? SS says: no.
 
 Contest landing page ---?
 
 Contest detail page ---?
+*** SS asks: what are these pages?
 
 I don't think (but will happily stand corrected) the run book needs to contain
 the 'why' -- only the 'how' and where/when/etc. of the process as experienced
 by the users/auditors. Each screenshot should have an associated index
 describing key features/fields/etc. which can be numerically identified.
 Instructions will instruct auditors to loop back if another round (or rounds)
 of ballots need to be inspected.
+***SS agrees.
 
 Besides a printed document with screenshots and instructions, we could also
 offer to create a tutorial video (videos, probably) where we film people using
 the tool/dashboards, with narration describing what they are doing, and what comes
 next depending on what’s happening.
+***SS says, it would be nice, but out of scope. We're so far over budget already....
+
+Brief introduction to RLAs in non-technical, human friendly language.
+At the county level, something along the lines of "This tool is going
+to help you cross reference votes recorded on paper ballots with how 
+the computers read those ballots, to determine whether the computers
```

Comment:
"how the system interpreted those ballots". Remember, interpretation is not purely done by computers. Human adjudication is part of the system.